Gemfile

```diff
@@ -19,7 +19,9 @@ gem 'bundler', '>= 1.3.0'
 gem 'railties', '~> 7.0.4'
 # # RAILS DEPENDENCIES
 
-gem 'sanger_warren'
+gem 'bunny', '>= 2.22.0'
+gem 'rbtree', '~> 0.4.5'
+gem 'sanger_warren', '~> 0.4.1'
```

Comment:
Is there a repo for sanger_warren ? Where does this come from ?

Reply:
[`sanger_warren`](https://rubygems.org/gems/sanger_warren/versions/0.4.1?locale=en) is available in RubyGems 😄